snooty.toml

```diff
@@ -21,6 +21,8 @@ toc_landing_pages = [
   "/telemetry"
   ]
 
+eol = true
```

Comment:
I would put this on line 3 because it says it should be at the top of the file. I'm not POSITIVE that this won't work as-is, but I know the placement matters so just to be safe, that's where I've always put it